src/programs/ectrans-benchmark.F90

```diff
@@ -194,6 +194,7 @@ program ectrans_benchmark
 integer(kind=jpim) :: jend_uder_EW = 0
 integer(kind=jpim) :: jbegin_vder_EW = 0
 integer(kind=jpim) :: jend_vder_EW = 0
+integer(kind=jpim) :: nresol1, nresol2
```

Comment:
just adding a comment here such that we don't forget to revert this file before merging.